contrib/extprotocol/init_file

```diff
@@ -0,0 +1,8 @@
+-- This file contains global patterns of messages that should be ignored or
+-- masked out, when comparing test results with the expected output.
+-- Individual tests can contain additional patterns specific to the test.
+
+-- start_matchsubs
+m/ERROR:  internal error: demoprot called with a different protocol.*/
+s/ERROR:  internal error: demoprot called with a different protocol.*/ERROR:  internal error: demoprot called with a different protocol (demoprot_new) (gpextprotocol.c:XXX)/
```

Comment:
fixed by remove lineno: https://code.hashdata.xyz/cloudberry/cbdb/-/commit/2f6e65cb7176c4f0b8336530ff39e44909a9266c

Reply:
Okay. Let me cherry-pick it.

Reply:
emmm... After I rechecked that commit. I have a second thought...
I think my way is the better way to solve that problem? There are actually two different lines report the same error. Isn't that a good idea to keep the line number?
@roseduan

Reply:
this import too many ignore in  init_file. But we need to change the same print out.

Reply:
Sorry, I just made another mistakes. It has not been merged into main branch yet... If 1X_stable is not necessary need to backport to main branch. Let me discuss with rose first... I still think my way is correct.

Reply:
Will pick to master, think about why has too many init_file, that's not follow rule  that print out should not the same and no line number.

Reply:
After the #101 is merged, I will cherry-pick it.

Reply:
cherry-picked.